Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocRowRenderer.cs

```diff
@@ -188,6 +205,48 @@ private IBocIndexColumnRenderer GetIndexColumnRenderer ()
       return _indexColumnRenderer;
     }
 
+    private void RenderValidationRow (
+        BocListRenderingContext renderingContext,
+        BocListRowRenderingContext rowRenderingContext,
+        BocListDataRowRenderEventArgs rowRenderEventArgs,
+        IReadOnlyCollection<BocListValidationFailureWithLocationInformation> validationFailuresWithLocationInfo,
+        int rowIndex)
+    {
+      var cssClassValidationRow = GetCssClassValidationRow(rowRenderingContext.IsSelected, rowRenderEventArgs);
+
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, cssClassValidationRow);
+      renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Tr);
+
+      var columnRenderers = renderingContext.ColumnRenderers;
+
+      var validationColumnRenderer = columnRenderers.First(r => r.ColumnDefinition is BocValidationErrorIndicatorColumnDefinition);
```

Comment:
possible alternative. Not sure which is better.
```suggestion
      var validationColumnRenderer = columnRenderers.OfType<IBocValidationErrorIndicatorColumnRenderer>().First();
```